setup.py

```diff
@@ -27,7 +27,7 @@
 
 AVRO_REQUIRES = ['fastavro>=0.23.0,<1.0;python_version<"3.0"',
                  'fastavro>=1.0;python_version>"3.0"',
-                 'avro==1.10.0'
+                 'avro==1.11.1',
```

Comment:
```suggestion
                 'avro>=1.11.1,<2.0.0',
```
Unless there's some reason we need to use 1.11.1, It's much better if you use a range strategy for specifying dependencies so that users of this library have more freedom to update their dependencies independently.

For example, I'm currently in a scenario where I am stuck using `avro==1.10.0` because that is the currently pinned version of the library. Unfortunately, there is a bug in version 1.10.0 that was fixed in 11.1.1. So now I need to wait for this library to update its dependencies in order to have my code function properly.

Reply:
:+1: